testing/base.py

```diff
@@ -1,74 +1,73 @@
+import os
+import shutil
 import unittest
-from importlib import reload
-import subprocess
 
 from six import with_metaclass
 
-from .mocks import SphinxAppMock
-from .utils import cd, deep_update
+import pytest
+
+
+TEST_DOC_DIR = os.path.join(os.path.dirname(__file__), 'doc')
+
+
+def make_default_config(project):
+    """
+    Creates a default configuration for exhale
+    """
+    return {
```

Comment:
Some parameters could actually be set in the minimal ``doc/conf.py``, but:
1/ it's good to have all relevant parameters at hand in one place
2/ it's future proof, we'll definitely need something like that for multi-project testing

Reply:
Yeah I like this approach, I just merged in my stashed changes with yours.  I was originally setting this in the metaclass, but it warrants a full method for sure.